php-checks/src/test/resources/checks/RandomGeneratorCheck.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+$random = rand(); // Noncompliant {{Use a cryptographically strong random number generator instead.}}
+//        ^^^^^^
+
+$random2 = mt_rand(0, 99); // Noncompliant
+//         ^^^^^^^^^^^^^^
+
+$random = random_int();
+$random = random_bytes(128);
+$random = openssl_random_pseudo_bytes(96);
```

Comment:
nitpick: add `gmp_random_bits` as compliant, too?

Reply:
I have no idea if GMP library is good for crypto, so I would avoid using it as a compliant example